mdbook/src/08-inputs-and-outputs/polling-sucks.md

```diff
@@ -0,0 +1,75 @@
+# Polling sucks, actually
+
+Oh yeah, turn signals usually blink, right?  How could we extend our program to blink the turn signal LED when a button is pressed.  We know how to blink an LED from our Hello World program; we turn on the LED, wait for some time, and then turn it off.  But how can we do this in our main loop while also checking for button presses?  We could try something like this:
+
+```rust
+    loop {
+        if button_a.is_low().unwrap() {
+            // Blink left arrow
+            display.show(&LEFT_ARROW);
+            timer.delay_ms(500_u32);
+            display.show(&BLANK);
+            timer.delay_ms(500_u32);
+        } else if button_b.is_low().unwrap() {
+            // Blink right arrow
+            display.show(&RIGHT_ARROW);
+            timer.delay_ms(500_u32);
+            display.show(&BLANK);
+            timer.delay_ms(500_u32);
+        } else {
+            display.show(&BLANK);
+        }
+        timer.delay_ms(10_u32);
+    }
+```
+
+Can you see the problem?  We're trying to do two things at once here: 
+
+1. Check for button presses
+2. Blink the LED
+
+But the processor can only do one thing at a time.  If we press a button during the blink delay, the processor won't be able to respond until the delay is over and the loop starts again.  As a result, we get a much less responsive program (try for yourself and see how worse it is).
```

Comment:
worse than what? Not checking for button presses? But that's a different thing.

Reply:
Thanks. Will replace with "see how much worse the button lag is".